codegen/consts.js

```diff
@@ -2,42 +2,66 @@ module.exports.props = require("react-html-attributes");
 module.exports.voids = ["area", "base", "br", "col", "embed", "hr", "img", "input", "link", "meta", "param", "source", "track", "wbr"];
 module.exports.types = {
   "allowFullScreen": "Boolean",
+  "allowTransparency": "Boolean",
   "async": "Boolean",
+  "autoComplete": "Boolean",
+  "autoFocus": "Boolean",
   "autoPlay": "Boolean",
   "capture": "Boolean",
+  "cellPadding": "Number",
+  "cellSpacing": "Number",
   "checked": "Boolean",
   "children": "Array JSX",
   "cols": "Number",
+  "colSpan": "Number",
+  "contentEditable": "Boolean",
   "controls": "Boolean",
+  "dangerouslySetInnerHTML": "{ __html :: String }",
   "default": "Boolean",
   "defer": "Boolean",
   "disabled": "Boolean",
+  "draggable": "Boolean",
   "formNoValidate": "Boolean",
+  "frameBorder": "Number",
   "hidden": "Boolean",
   "itemScope": "Boolean",
   "loop": "Boolean",
+  "marginHeight": "Number",
+  "marginWidth": "Number",
+  "maxLength": "Number",
   "max": "Number",
+  "minLength": "Number",
   "min": "Number",
   "multiple": "Boolean",
   "muted": "Boolean",
+  "noValidate": "Boolean",
+  "onBlur": "EventHandler",
+  "onChange": "EventHandler",
   "onClick": "EventHandler",
+  "onFocus": "EventHandler",
   "onInput": "EventHandler",
   "onInvalid": "EventHandler",
+  "onKeyDown": "EventHandler",
+  "onKeyPress": "EventHandler",
+  "onKeyUp": "EventHandler",
   "onSubmit": "EventHandler",
-  "noValidate": "Boolean",
-  "onChange": "EventHandler",
   "open": "Boolean",
   "playsInline": "Boolean",
   "readOnly": "Boolean",
   "required": "Boolean",
   "reversed": "Boolean",
-  "rowSpan": "Number",
   "rows": "Number",
+  "rowSpan": "Number",
```

Comment:
MDN lists `rowspan` and `colspan` props as integers. Any reason not to use `Int` instead?

Reply:
All of the numerical types there are currently Number rather than Int, though, even though many of the others appear to have to be integers.  I have made my additions consistent with what is there already in this regard.

If these are made Int, then perhaps the previously supported attributes should be reviewed (and maybe many changed to Int?)

What's your preference?

Reply:
I see, hmm. I think this needs to be a major update anyway so it might be good to fix them all. But only ones which are actually documented as integer-only.

Reply:
I can do that after this gets merged though

Reply:
OK, I'll go through and review all the Number types against the spec, and update the PR.